### ISSUE TITLE
Font chosen not converting to saved file

### ISSUE THREAD

**Issue by bksmth** [lettier/gifcurry#60]
Hey, not a coder here

Running Gifcurry 6.0.1.0 on Unbuntu 20.04

This could be an issue with just the one font but Uroob Italic, Uroob Bold and Uroob Bold Italic all becomes Uroob Regular when saving the file. I've tried messing around with the setting in the .yaml files but that doesn't seem to change anything.
![issue gifcurry](https://user-images.githubusercontent.com/74503781/99199633-810e0e80-276e-11eb-9f60-0c3f13c1ce57.png)

Thanks and have a great day
-Brian

**Comment by lettier** [comment]
Hello,

When you run Gifcurry from the terminal, what is the output under `TEXT > Text > Font > Family`?

```yaml
  - TEXT:
    - Text: 
      - Font:
        - Family: ?
```

If you run

```bash
convert -list font
```
can you find the font in the output?

:+1:

**Comment by bksmth** [comment]
Here is a screenshot from my terminal showing the output when I tried recreating that gif file. It came out the same as my original post even though 'Style: Italic' is in that output.

![issue gifcurry term output](https://user-images.githubusercontent.com/74503781/99393823-e7e51200-28ab-11eb-8cae-ca29b072fb1b.png)

When I ran

   `convert -list font`

I saw that Uroob is not listed but also that the number of listed fonts was not nearly as many as are available to choose from in Gifcurry GUI.
